src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -95,7 +95,39 @@ class PathManager : IPathManager
 
         private IEnumerable<string> RootDirectories
         {
-            get { return Preferences != null ? Preferences.CustomPackageFolders.Where(path => path != DynamoModel.StandardLibraryToken) : rootDirectories; }
+            get 
+            { 
+                return Preferences != null ? 
+                    Preferences.CustomPackageFolders.Select(path => path == DynamoModel.StandardLibraryToken ? StandardLibraryDirectory : path) 
+                    : rootDirectories;
```

Comment:
each item in CustomPackageFolders is tested if it equals StandardLibraryToken. Should we be testing if it contains the token (contains not equals) ?

Reply:
I think it will exactly equal the token, I can't think of a scenario where a substring of the path will match the token.